src/bpftrace.h

```diff
@@ -145,6 +151,7 @@ class BPFtrace
   std::vector<std::string> join_args_;
   std::vector<std::string> time_args_;
   std::vector<std::tuple<std::string, std::vector<Field>>> cat_args_;
+  std::map<int64_t, struct HelperErrorInfo> helper_error_info_;
```

Comment:
```suggestion
  std::unordered_map<int64_t, struct HelperErrorInfo> helper_error_info_;
```
Do we need the ordering? unordered_map should be faster lookups